OpenEdXMobile/res/layout/activity_register.xml

```diff
@@ -3,120 +3,135 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:background="@android:color/white"
     android:orientation="vertical"
-    android:paddingBottom="8dp"
-    android:background="@android:color/white">
+    android:paddingBottom="8dp">
```

Comment:
`@dimen/vertical_input_padding` would be better i think.
`@dimen/widget_margin` has value of `10dp`